iron-fit-behavior.html

```diff
@@ -123,8 +133,18 @@
       },
 
       /**
-       * The same as setting margin-top and margin-bottom css properties.
-       * @deprecated
```

Comment:
Not really, before this PR the behavior was "it will override the margin of your element", with this PR it doesn't do it anymore. The user that was counting on the previous behavior is a user that wouldn't set the margins of the element, since this would be done by the behavior. Default margins are 0 thankfully, so in short with this PR everyone should gain 😁